pip/req/req_install.py

```diff
@@ -727,7 +727,11 @@ def uninstall(self, auto_confirm=False):
 
         # find console_scripts
         if dist.has_metadata('entry_points.txt'):
-            config = configparser.SafeConfigParser()
+            if six.PY2:
+                options = {}
+            else:
+                options = {"delimiters": ('=')}
```

Comment:
of course... Me dumm! I'm not used to use tuples. Thank for spotting it. Committing